update(platform): removing storage service availability impact (#139)
aiven/aiven-docs: docs/platform/howto/add-storage-space.md

:::

> ## Update service storage via console

Reviewer: ```suggestion
## Update service storage via the console
```

1. Click **Save Changes**.

If you added storage, the additional storage is available for immediate use.
If you removed additional storage, the service nodes go through a rolling restart

> accompanied by a minimal downtime.

Reviewer 2: ```suggestion
which may be accompanied by a minimal downtime depending on service type and plan tier.
```

1. Click **Save Changes**.

If you added storage, the additional storage is available for immediate use.
If you removed additional storage, the service nodes go through a rolling restart

> which may be accompanied by a minimal downtime, depending on the service type and the plan

Reviewer 2: ```suggestion
which may be accompanied by a minimal downtime for services that don't have HA capabilities, this is dependent on the service type and the plan
```

> Depending on the service type and plan, such a restart may be accompanied by a minimal
> downtime for services with no HA capabilities.

Reviewer: We can reduce word count a bit:
```suggestion
Depending on the service type and plan, there might be short
downtime for services with no HA capabilities.
```
